README.md

```diff
@@ -51,13 +51,18 @@ The `exec` function takes some options, with these defaults:
   isBuffer: false,
   shell: undefined,
   logger: undefined,
+  maxStdoutBufferSize: 20 * 1024 * 1024, // 20 MB
+  maxStdoutBufferSize: 20 * 1024 * 1024, // 20 MB
```

Comment:
i don't see any reason to repeat the same key, did you mean `Stderr` perhaps?

Reply:
it is a copypaste issue, good catch